pages/_app.tsx

```diff
@@ -1,8 +1,47 @@
-import '../styles/globals.css'
+
 import type { AppProps } from 'next/app'
+import React, { useMemo, useState } from 'react';
+import { createTheme, CssBaseline, ThemeProvider, useMediaQuery } from '@mui/material';
+import { ColorModeContext } from '../utils/ColorModeContext';
 
 function MyApp({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />
+  const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');
```

Comment:
Get the user's browser preference and use this for the default

Reply:
This is cool, I wouldn't have thought to do this